content/en/docs/concepts/services-networking/dns-pod-service.md

```diff
@@ -105,7 +105,7 @@ Kube-DNS versions, prior to the implementation of the
 had the following DNS resolution:
 
 ```
-pod-ipv4-address.my-namespace.pod.cluster-domain.example
+pod-ipv4-address.namespace.pod.cluster-domain.example
```

Comment:
This may be a confusing change as the intention to convey that the Pod's DNS has a reference to the namespace the Pod is in, is no longer there. 
I understand there is an example added following this line